src/components/AnalysisModal.js

```diff
@@ -276,10 +292,11 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
             tooltip: Utils.summarizeErrors(errors),
             onClick: async () => {
               try {
-                const contents = isJupyter ? notebookData[notebookKernel] : '# Starter Rmd file'
+                const contents = isJupyter ? notebookData[notebookKernel] : '# Starter R file'
+                const fullAnalysisName = `${analysisName}.${isJupyter ? tools.Jupyter.ext : fileExt}`
```

Comment:
The `isJupyter` case looks like it only works if `tools.Jupyter.ext` has one item. Should this be `tools.Jupyter.defaultExt`?